staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -184,6 +185,30 @@ func NewConflict(qualifiedResource schema.GroupResource, name string, err error)
 	}}
 }
 
+// NewApplyConflict returns an error including details on the requests apply conflicts
+func NewApplyConflict(conflicts merge.Conflicts, err error) *StatusError {
+	causes := make([]metav1.StatusCause, 0, len(conflicts))
+	for i := range conflicts {
+		err := conflicts[i]
```

Comment:
This is shadowing the other `err`, not sure this is good practice.

Reply:
No it isn't.